.eslintrc.js

```diff
@@ -10,4 +10,38 @@ module.exports = {
     'no-useless-escape': 'off',
     'no-case-declarations': 'off',
   },
+  overrides: [
+    {
+      files: ['./templates/demo-store/*'],
```

Comment:
This was included to ensure that the pre-commit scripts for eslint used the correct `import/order` definition since the lint configuration at the root of the monorepo will be used for the eslint check.